src/utils/hooks/useLoadApplication.tsx

```diff
@@ -30,7 +30,7 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
       serial: serialNumber,
     },
     skip: triggerProcessing || isApplicationLoaded,
-    fetchPolicy: networkFetch ? 'no-cache' : 'cache-first',
+    fetchPolicy: networkFetch ? 'network-only' : 'cache-first',
```

Comment:
Cool, I'm glad this works. I remember trying to use `network-only` before and it just not working, so I left it as `no-cache`. No idea why, but glad it seems to be working now.